admin/tabs/monitor/MonitorTabModel.ts

```diff
@@ -73,7 +74,7 @@ export class MonitorTabModel extends HoistModel {
     }
 
     override async doLoadAsync(loadSpec: LoadSpec) {
-        if (!XH.pageIsVisible) return;
```

Comment:
FWIW we are showing a toast out of this model when the status changes - with the idea that you might not have this tab visible (i.e. are on new clients tab, or configs) but would still appreciate a heads up that a monitor just ticked over. Might want to keep prior viz check if we thought that worthwhile